examples/misc/lib/cheatsheet/redirecting_constructors.dart

```diff
@@ -17,11 +17,11 @@ class Automobile {
 
 // #docregion const-constructors
 class ImmutablePoint {
-  const ImmutablePoint(this.x, this.y);
+  static const ImmutablePoint origin = ImmutablePoint(0, 0);
 
   final int x;
   final int y;
 
-  static const ImmutablePoint origin = ImmutablePoint(0, 0);
+  const ImmutablePoint(this.x, this.y);
```

Comment:
I'm curious why the constructor is now last? I usually put the constructors at the top of a class declaration.

Reply:
I guess this is just the form I've seen used most commonly throughout non-Flutter Dart code and on the website, but I'm happy to switch back. Just let me know :)

Reply:
I'd suggest landing this as is, and then we can figure out the style guide issues =)

Hey @RedBrogdon, do we have a style guide somewhere on constructor placement in Dart classes?